### PR TITLE
fix: Uio enhance inputs mobile button (resolves #243)

### LINKED ISSUE #243: UIO buttons on mobile layout are broken when UIO "enhance inputs" is enabled
## Describe the bug

When UIO "enhance inputs" is enabled, part of UIO buttons "show preferences", "hide preferences" become invisible.

## To Reproduce

1. Switch the browser to use responsive design mode. Pick a screen size that shows mobile layout, for example "iphone 5";
2. Open UIO and enable "enhance inputs";
3. The top and bottom parts of "show preferences", "hide preferences" become invisible.

## Expected behavior

Full button texts should be visible.

## Screenshots

Before "enhance inputs" is enabled:
Screen Shot 2020-05-27 at 11 15 16 AM

After "enhance inputs" is enabled:
Screen Shot 2020-05-27 at 11 15 26 AM

## Additional context

Increasing the height of the button container might fix the problem.

### PR DESCRIPTION
**Pull request by TeddTech**
* [x] This pull request has been linted by running `npm run lint` without errors
* [x] This pull request has been tested by running `npm run start` and reviewing affected routes
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description
When UIO "enhance inputs" is enabled the UIO tab is now fully visible in mobile view.

## Steps to test

1. Go to deploy preview
2. View in mobile mode
3. Look at UIO tab at top of page

**Expected behavior:** 

UIO tab buttons should now be fully visible.
## Additional information
N/A

## Related issues

resolves #243

### CODE REVIEW

**Comment by cindyli** [comment]
Today's check-in meeting decides to increase the space between "hide preferences" and "reset" buttons on the mobile view so they look more like separate buttons.

**Comment by TeddTech** [comment]
> Today's check-in meeting decides to increase the space between "hide preferences" and "reset" buttons on the mobile view so they look more like separate buttons.

I will also be removing the shadow from underneath the uio-tab in mobile view in this PR

**Comment by TeddTech** [comment]
Still working on IPad Pro view

**Comment by TeddTech** [comment]
Ready for review

**Comment by cindyli** [comment]
When enabling "enhance inputs" and setting the UIO text styling to "Comic Sans" or "Verdana", the line underneath the "reset" button becomes invisible.

Screen Shot 2020-06-03 at 10 07 41 AM

**Comment by cindyli** [comment]
When enabling "enhance inputs" and setting the UIO text styling to "Verdana", on small screens such as "iphone 5", the text "hide preferences" gets pushed out of the screen which creates the horizontal overflow.

Screen Shot 2020-06-03 at 10 20 11 AM

File: src/scss/vendors/_vendors.scss [hunk 1]
```diff
@@ -31,6 +31,59 @@
...
+	}
+
+	button[aria-expanded="true"] {
+		border-bottom: 2px solid #ccc;
+		margin-left: rem(8);
+		padding-left: rem(10);
+	}
+}
+
+@media screen and (min-width: 40em) {
```
Comment (cindyli) [inline src/scss/vendors/_vendors.scss:60]: Add a code comment to explain this media query breakpoint comes from the infusion preferences framework for UIO in order to customize UIO button styles on desktop and mobile.

File: src/scss/vendors/_vendors.scss [hunk 2]
```diff
@@ -33,34 +33,39 @@
...
+		width: 97%;
 	}
 }
 
-@media screen and (min-width: 40em) {
+@media screen and (min-width: 40em) { // This media query breakpoint comes from the infusion preferences framework for UIO in order to customize UIO button styles on desktop and mobile.
 	.fl-input-enhanced #defaultContainer header .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
 		display: block;
 		height: auto;
+		padding: 0.625rem 1.25rem;
```
Comment (cindyli) [inline src/scss/vendors/_vendors.scss:68]: Please use `rem()` function.

**Pull request merged**